newsletter-april-2021.rst

```diff
@@ -0,0 +1,107 @@
+.. post:: April 5, 2021
+   :tags: newsletter
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from April.
+
+Read the Docs newsletter - April 2021
+=====================================
+
+This is the first of our monthly newsletters, in which we would like to
+openly share with you the most relevant updates of Read the Docs,
+offer a summary of what new features we shipped to our users
+during the previous month,
+and share what things we will be focusing on in the near future.
+
+Company highlights
+------------------
+
+-  We have a new colleague! `Juan Luis <https://blog.readthedocs.com/archive/author/juan-luis-cano-rodriguez/>`_
+   will be working with us as Developer Advocate, with a focus on fulfilling
+   the goals of `the CZI grant we were
+   awarded <https://blog.readthedocs.com/czi-grant-announcement/>`_,
+   improve our public facing documentation,
+   and spread the word about our service.
+-  xxx we saw an increment of X % page views
```

Comment:
This should be updated before publishing

Reply:
Wooops good catch.

Reply:
Updated